ui/src/test-data/mock-up-issues.tsx

```diff
@@ -0,0 +1,242 @@
+/*
+ * Copyright (C) 2024 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+// Some mock-up data to test the functionality of the Issues data table.
+// Data was created with ChatGPT, and for most part, includes made up
+// random data, not reflecting the real data that would be returned from
+// an ORT run.
+
+import { Severity } from '@/api/requests';
+
+export const mockUpIssues = {
```

Comment:
As we don't have this sort of data anywhere else, I wouldn't commit it here either.